src/main/java/io/strimzi/kafka/bridge/SinkBridgeEndpoint.java

```diff
@@ -652,4 +652,28 @@ protected void commit(Map<TopicPartition, io.vertx.kafka.client.consumer.OffsetA
             }
         });
     }
+
+    protected void seek(TopicPartition topicPartition, long offset, Handler<AsyncResult<Void>> seekHandler) {
+        this.consumer.seek(topicPartition, offset, result -> {
+            if (seekHandler != null) {
```

Comment:
Why would this method be called with a null `seekHandler`? Unlike the other methods in `SinkBridgeEndpoint` the parameter is not used to set a field.